core: add setting to open application launch URL in a new browser tab

frprm opened the pull request:
<!--
👋 Hello there! Welcome.

Please check the [Contributing guidelines](https://github.com/goauthentik/authentik/blob/main/CONTRIBUTING.md#how-can-i-contribute).
-->

# Details
* **Does this resolve an issue?** No

## Changes
### New Features
* Adds a setting to open application launch URL in a new browser tab

frprm committed:
core: add setting to open application launch URL in a new browser tab

BeryJu reviewed (commented):
Cheers for the PR! Looks pretty good, could you set `debug: true` in your local config and re-run `make gen-build`? Some of the models are only loaded when debug mode is enabled and as such, debug mode has to be enabled for generating the scheme.
Also I think there might be some small formatting things, run `make` and `make web` to autoformat them

frprm committed:
core: fix failing applications unit tests

frprm committed:
core: fix formatting

frprm committed:
core: include models only generated when debug mode is enabled

frprm commented:
Thanks for taking a look at my PR! I think I've addressed the concerns you shared earlier with the commits I just pushed... Let me know if there's anything else.
